README.md

```diff
@@ -64,32 +64,38 @@ git checkout -b <your_branch_name>
 > 1. create a folder and add all code in that folder only.
 > 2. create a demo folder inside "Your Project" and add a gif of your project named demo.gif
 
-**9.** Track your changes:heavy_check_mark: .
+**9.** If you want, you can host the website locally in your machine to see the changes in real time.
```

Comment:
I feel the changes in README.md are not needed as such. In fact, thy could just be more confusing for the contributors